[2022-02-27 12:58:11]       127.0.0.1 INFO: Log system.log Initialized.  
[2022-03-14 00:34:33]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:34:33]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:34:33]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 00:34:56]  192.168.77.118 WARNING: User attempted to access area without proper permissions {"Type":"DvlpUpdate","IP":"192.168.77.118"} 
[2022-03-14 00:39:35]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:39:35]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:39:35]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 00:39:42]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:39:42]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:39:42]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 00:40:18]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:40:18]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:40:18]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 00:41:23]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:41:23]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:41:23]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 00:51:32]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:51:32]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 00:51:32]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 01:24:47]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 01:24:47]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-14 01:24:47]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-14 20:25:03]  192.168.77.118 INFO: An exception occurred while executing 'UPDATE gl_quizzer_quizzes SET quizID = :quizID, quizName = :quizName, introtext = :introtext, introfields = :introfields, pass_msg = :pass_msg, fail_msg = :fail_msg, enabled = :enabled, fill_gid = :fill_gid, onetime = :onetime, questionsAsked = :asked, levels = :levels WHERE quizID = :old_id' with params ["q1", "q1", "Test Quiz QT1", "Partial credit is given for multiple-check questions.", "Name|Department", "You have passed the quiz", "You have failed the quiz", 1, 2, 0, 0, "55"]: Value for :asked not found in params array. Params array key should be "asked"  
[2022-03-14 20:25:07]  192.168.77.118 INFO: An exception occurred while executing 'UPDATE gl_quizzer_quizzes SET quizID = :quizID, quizName = :quizName, introtext = :introtext, introfields = :introfields, pass_msg = :pass_msg, fail_msg = :fail_msg, enabled = :enabled, fill_gid = :fill_gid, onetime = :onetime, questionsAsked = :asked, levels = :levels WHERE quizID = :old_id' with params ["q1", "q1", "Test Quiz QT1", "Partial credit is given for multiple-check questions.", "Name|Department", "You have passed the quiz", "You have failed the quiz", 1, 2, 0, 0, "55"]: Value for :asked not found in params array. Params array key should be "asked"  
[2022-03-14 20:25:31]  192.168.77.118 INFO: An exception occurred while executing 'UPDATE gl_quizzer_quizzes SET quizID = :quizID, quizName = :quizName, introtext = :introtext, introfields = :introfields, pass_msg = :pass_msg, fail_msg = :fail_msg, enabled = :enabled, fill_gid = :fill_gid, onetime = :onetime, questionsAsked = :asked, levels = :levels WHERE quizID = :old_id' with params ["q1", "q1", "Test Quiz QT1", "Partial credit is given for multiple-check questions.", "Name|Department", "You have passed the quiz", "You have failed the quiz", 1, 2, 0, 0, "55"]: Value for :asked not found in params array. Params array key should be "asked"  
[2022-03-14 20:29:34]  192.168.77.118 ERROR:   
[2022-03-15 15:56:51]  192.168.77.118 ERROR: array ( 'action' => 'toggleSub', 'uid' => '2', 'oldval' => '0', )  
[2022-03-15 16:43:11]  192.168.77.118 ERROR: changing from 1  
[2022-03-15 16:43:35]  192.168.77.118 ERROR: changing from 0  
[2022-03-15 16:43:52]  192.168.77.118 ERROR: changing from 1  
[2022-03-15 16:43:54]  192.168.77.118 ERROR: changing from 0  
[2022-03-15 16:48:01]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '2', 'oldval' => '1', )  
[2022-03-15 16:48:01]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 2, 'month' => 1, 'day' => 4, 'sendcards' => 0, ))  
[2022-03-15 16:48:47]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '2', 'oldval' => '0', )  
[2022-03-15 16:48:47]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 2, 'month' => 1, 'day' => 4, 'sendcards' => 0, ))  
[2022-03-15 16:48:47]  192.168.77.118 ERROR: unsubscribing  
[2022-03-15 16:49:32]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '2', 'oldval' => '0', )  
[2022-03-15 16:49:32]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 2, 'month' => 1, 'day' => 4, 'sendcards' => 0, ))  
[2022-03-15 16:49:32]  192.168.77.118 ERROR: unsubscribing  
[2022-03-15 17:11:02]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 17:11:02]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 17:11:02]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-15 17:27:24]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Regular' ' with params ["Regular"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:27:24]  192.168.77.118 ERROR: Class "Membership\Databae" not found  
[2022-03-15 17:28:35]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Family' ' with params ["Family"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:35]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Inactive' ' with params ["Inactive"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:35]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Junior' ' with params ["Junior"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:35]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Regular' ' with params ["Regular"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Regular' ' with params ["Regular"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: Class "Membership\Databae" not found  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:57]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT * FROM gl_membership_plans WHERE plan_id='Regular' ' with params ["Regular"]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: Class "Membership\Databae" not found  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'SELECT u.*, m.terms_accept FROM gl_users u LEFT JOIN gl_membership_users m ON m.uid = u.uid WHERE u.uid=2' with params [2]: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens  
[2022-03-15 17:28:58]  192.168.77.118 ERROR: An exception occurred while executing 'UPDATE gl_pm_userprefs SET uid = ?, block = ?, notify = ?' with params [2, 0, 1]: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2' for key 'PRIMARY'  
[2022-03-15 17:39:42]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '0', )  
[2022-03-15 17:39:42]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 0, ))  
[2022-03-15 17:39:42]  192.168.77.118 ERROR: subscribing  
[2022-03-15 17:39:42]  192.168.77.118 ERROR: subbing  
[2022-03-15 17:40:01]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '0', )  
[2022-03-15 17:40:01]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 0, ))  
[2022-03-15 17:40:01]  192.168.77.118 ERROR: subscribing  
[2022-03-15 17:40:01]  192.168.77.118 ERROR: subbing  
[2022-03-15 17:40:26]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '0', )  
[2022-03-15 17:40:26]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 0, ))  
[2022-03-15 17:40:26]  192.168.77.118 ERROR: subscribing  
[2022-03-15 17:40:26]  192.168.77.118 ERROR: subbing  
[2022-03-15 17:40:26]  192.168.77.118 ERROR: subbing  
[2022-03-15 17:40:43]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '0', )  
[2022-03-15 17:40:43]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 0, ))  
[2022-03-15 17:40:43]  192.168.77.118 ERROR: subscribing  
[2022-03-15 17:41:18]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '0', )  
[2022-03-15 17:41:18]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 0, ))  
[2022-03-15 17:41:18]  192.168.77.118 ERROR: subscribing  
[2022-03-15 17:41:18]  192.168.77.118 ERROR: value is now 1  
[2022-03-15 17:41:18]  192.168.77.118 ERROR: array ( 'uid' => '3', 'newval' => true, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:41:21]  192.168.77.118 ERROR: array ( 'action' => 'toggleCards', 'uid' => '3', 'oldval' => '1', )  
[2022-03-15 17:41:21]  192.168.77.118 ERROR: Birthdays\Birthday::__set_state(array( 'uid' => 3, 'month' => 10, 'day' => 29, 'sendcards' => 1, ))  
[2022-03-15 17:41:21]  192.168.77.118 ERROR: unsubscribing  
[2022-03-15 17:46:08]  192.168.77.118 ERROR: changing to 1  
[2022-03-15 17:46:08]  192.168.77.118 ERROR: value is now 1  
[2022-03-15 17:46:08]  192.168.77.118 ERROR: array ( 'uid' => '3', 'newval' => true, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:46:12]  192.168.77.118 ERROR: changing to 0  
[2022-03-15 17:46:12]  192.168.77.118 ERROR: value is now   
[2022-03-15 17:46:12]  192.168.77.118 ERROR: array ( 'uid' => '3', 'newval' => false, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:47:07]  192.168.77.118 ERROR: changing to 1  
[2022-03-15 17:47:07]  192.168.77.118 ERROR: value is now 1  
[2022-03-15 17:47:07]  192.168.77.118 ERROR: array ( 'uid' => '3', 'newval' => 1, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:47:12]  192.168.77.118 ERROR: changing to 0  
[2022-03-15 17:47:12]  192.168.77.118 ERROR: value is now 0  
[2022-03-15 17:47:12]  192.168.77.118 ERROR: array ( 'uid' => '2', 'newval' => 0, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:53:21]  192.168.77.118 ERROR: changing to 1  
[2022-03-15 17:53:21]  192.168.77.118 ERROR: value is now 1  
[2022-03-15 17:53:21]  192.168.77.118 ERROR: array ( 'uid' => '2', 'newval' => 1, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 17:53:22]  192.168.77.118 ERROR: changing to 0  
[2022-03-15 17:53:22]  192.168.77.118 ERROR: value is now 0  
[2022-03-15 17:53:22]  192.168.77.118 ERROR: array ( 'uid' => '2', 'newval' => 0, 'statusMessage' => 'Subscription has been updated', )  
[2022-03-15 18:51:07]  104.168.65.222 ERROR: An exception occurred while executing 'SELECT 2016 as year, b.*, u.username, u.fullname FROM gl_birthdays b LEFT JOIN gl_users u ON u.uid=b.uid WHERE b.month = :month': SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':month' at line 1  
[2022-03-15 19:15:30]   202.138.242.6 ERROR: An exception occurred while executing 'SELECT 2016 as year, b.*, u.username, u.fullname FROM gl_birthdays b LEFT JOIN gl_users u ON u.uid=b.uid WHERE b.month = :month': SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':month' at line 1  
[2022-03-15 19:49:59]    34.96.130.30 ERROR: An exception occurred while executing 'SELECT 2016 as year, b.*, u.username, u.fullname FROM gl_birthdays b LEFT JOIN gl_users u ON u.uid=b.uid WHERE b.month = :month': SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':month' at line 1  
[2022-03-15 19:57:34]       127.0.0.1 ERROR: An exception occurred while executing 'SELECT 2016 as year, b.*, u.username, u.fullname FROM gl_birthdays b LEFT JOIN gl_users u ON u.uid=b.uid WHERE (b.month = :month) AND (b.day = :day)': SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':month) AND (b.day = :day)' at line 1  
[2022-03-15 21:16:17]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 21:16:17]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 21:16:17]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-15 21:30:58]  192.168.77.118 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 52 Empty reply from server  
[2022-03-15 21:30:58]  192.168.77.118 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:30:58]  192.168.77.118 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/34.0536909,-118.242766?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:30:59]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 21:30:59]  192.168.77.118 ERROR: The given alias 'm' is not part of any FROM or JOIN clause table. The currently registered aliases are: gl_membership_members.  
[2022-03-15 21:30:59]  192.168.77.118 ERROR: Call to a member function rowCount() on int  
[2022-03-15 21:40:29]  170.83.179.127 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:40:29]  170.83.179.127 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:40:29]  170.83.179.127 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:40:34]  170.83.179.127 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 22 The requested URL returned error: 403 Forbidden  
[2022-03-15 21:40:34]  170.83.179.127 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:40:34]  170.83.179.127 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:45:02]   107.158.93.82 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:45:02]   107.158.93.82 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:45:02]   107.158.93.82 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:45:28]  196.242.46.203 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 28 Connection timed out after 10001 milliseconds  
[2022-03-15 21:45:28]  196.242.46.203 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:45:28]  196.242.46.203 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:45:32]   107.158.93.82 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 52 Empty reply from server  
[2022-03-15 21:45:32]   107.158.93.82 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:45:32]   107.158.93.82 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:45:32]  196.242.46.203 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:45:32]  196.242.46.203 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:45:32]  196.242.46.203 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:56:51]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:56:51]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:56:51]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:57:03]  186.179.17.147 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:57:03]  186.179.17.147 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:57:03]  186.179.17.147 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:57:05]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:57:05]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:57:05]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:57:18]  186.179.17.147 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 28 Connection timed out after 10001 milliseconds  
[2022-03-15 21:57:18]  186.179.17.147 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:57:18]  186.179.17.147 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:57:31]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 28 Connection timed out after 10001 milliseconds  
[2022-03-15 21:57:31]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:57:31]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 21:57:52]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 21:57:52]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 21:57:52]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:11:38]   23.105.159.92 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:11:38]   23.105.159.92 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:11:38]   23.105.159.92 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:11:55]   23.105.159.92 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:11:55]   23.105.159.92 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:11:55]   23.105.159.92 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:35:28] 138.186.137.198 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 52 Empty reply from server  
[2022-03-15 22:35:28] 138.186.137.198 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:35:28] 138.186.137.198 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:35:44] 138.186.137.198 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:35:44] 138.186.137.198 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:35:44] 138.186.137.198 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:38:15]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:38:15]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:38:15]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:38:30]   72.249.144.12 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:38:30]   72.249.144.12 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:38:30]   72.249.144.12 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:43:30]    114.4.98.150 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 56 Recv failure: Connection reset by peer  
[2022-03-15 22:43:30]    114.4.98.150 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:43:30]    114.4.98.150 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
[2022-03-15 22:43:47]    114.4.98.150 ERROR: WeatherUnlocked Weather\API::fetchWeather() Error: 28 Connection timed out after 10000 milliseconds  
[2022-03-15 22:43:47]    114.4.98.150 ERROR: WeatherUnlocked Empty weather data received.  
[2022-03-15 22:43:47]    114.4.98.150 ERROR: WeatherUnlocked ... Attempted to retrieve url http://api.weatherunlocked.com/api/forecast/40.9452911,-74.0735899?app_id=7b599d44&app_key=a759e3aac3486002ef2dfabebdf24f28  
